validators/__init__.py

```diff
@@ -13,6 +13,7 @@
 from .card import amex, card_number, diners, discover, jcb, mastercard, unionpay, visa
 from .domain import domain
 from .email import email
+from crypto_addresses.eth_address import eth_address
```

Comment:
Use `.` relative import, put it along with `.i18n` import statements.

```diff
-from crypto_addresses.eth_address import eth_address
+from .crypto_addresses import eth_address
```

Reply:
Done. [e4128c9](https://github.com/python-validators/validators/pull/276/commits/e4128c9943788e7a338ca72bb9552f1a129637ee)